metallb/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: metallb
+
+resources:
+  - https://raw.githubusercontent.com/metallb/metallb/v0.9.5/manifests/namespace.yaml
+  - https://raw.githubusercontent.com/metallb/metallb/v0.9.5/manifests/metallb.yaml
+  - clusterrole.yaml
+  - rolebinding.yaml
```

Comment:
Not present here anymore.

Reply:
Yes, I realize that. Still making changes.